### ISSUE TITLE
Dude, Where's My Tree?

### ISSUE THREAD

**Issue by antgonza** [qiyunzhu/woltka#3]
Following the gOTU example, the next section is using QIIME2 for analysis but is not clear where the tree being used comes from; [see](https://github.com/qiyunzhu/woltka/tree/dev#gotu-analysis-using-qiime-2). Additionally, looking around those instructions I couldn't find any complete information about this tree.

**Comment by qiyunzhu** [comment]
Good point! The tree is a resource downloadable from the WoL website, but as you said the instruction is vague for users who don't want to go thru the entire WoL website. Currently, a pre-compiled tree for QIIME 2 (type `Phylogeny[Rooted]`) is available from our data release, which is a Globus endpoint, and is not very straightforward. Having the file downloadable from the website should be favorable.

**Comment by antgonza** [comment]
Yeah, adding these lines to the documentation will be super helpful; thank you.
